string name  # ""
string window  # ""
string tab_name  # ""
bool remove  # False
# default values for all sliders, min and max are relative to these numbers
geometry_msgs/TransformStamped transform_stamped
float64 min  # -1.0
float64 max  # 1.0
